lib/flex_ui/widgets/rounded_image.dart

```diff
@@ -0,0 +1,42 @@
+import 'package:flex_storefront/flex_ui/tokens/sizes.dart';
+import 'package:flex_storefront/flex_ui/widgets/cached_image.dart';
+import 'package:flutter/material.dart';
+
+class RoundedImage extends StatelessWidget {
+  const RoundedImage({
+    super.key,
+    required this.imageUrl,
+    this.width,
+    this.height,
+    this.fit = BoxFit.contain,
+    this.borderRadius = FlexSizes.borderRadiusMd,
+    this.onPressed,
+  });
+
+  final String imageUrl;
+  final double? width, height;
```

Comment:
Can you declare it on 2 lines? I'm not seeing this often enough to be confortable with it lol